src/mds/SessionMap.h

```diff
@@ -224,6 +225,13 @@ class Session : public RefCountedObject {
     }
   }
 
+  bool is_session_flushing_msgs(version_t client_id) {
```

Comment:
clent_t client_id

Reply:
this function is completely wrong.  covert client to version does not make sense at all. you should just check if waitfor_flush is empty